modules/economy-system/commands/economy-system.js

```diff
@@ -0,0 +1,378 @@
+    },
+    'weekly': async function (interaction) {
+        const model = interaction.client.models['economy-system']['cooldown'];
+        const cooldownModel = await model.findOne({
+            where: {
+                userId: interaction.user.id,
+                command: 'weekly'
+            }
+        });
+        if (cooldownModel) {
+            return interaction.reply({
+                content: interaction.strings['cooldown'],
+                ephemeral: true
+            });
+        }
+        const cooldown = await model.create({
+            userId: interaction.user.id,
+            command: 'weekly'
+        });
+        balance(interaction.client, interaction.user.id, 'add', interaction.client.configurations['economy-system']['config']['weeklyReward']);
+        interaction.reply({
+            content: `You erned ${interaction.client.configurations['economy-system']['config']['weeklyReward']} ${interaction.client.configurations['economy-system']['config']['currencySymbol']} by claiming your weekly reward`,
+            ephemeral: true
+        });
+        setTimeout(async () => {
+            await cooldown.destroy();
+        }, 604800000);
+        interaction.client.logger.info(`[economy-system] The user ${interaction.user.username}#${interaction.user.discriminator} gained ${interaction.client.configurations['economy-system']['config']['dailyReward']} ${interaction.client.configurations['economy-system']['config']['currencySymbol']} by claiming the weekly reward`);
+        if (interaction.client.logChannel) interaction.client.logChannel.send(`[economy-system] The user ${interaction.user.username}#${interaction.user.discriminator} gained ${interaction.client.configurations['economy-system']['config']['dailyReward']} ${interaction.client.configurations['economy-system']['config']['currencySymbol']} by claiming the weekly reward`);
+    },
+    'balance': async function (interaction) {
+        let user = interaction.options.getUser('user');
+        if (!user) user = interaction.user;
+        const balanceV = await interaction.client.models['economy-system']['Balance'].findOne({
+            where: {
+                id: user.id
+            }
+        });
+        if (!balanceV) return interaction.reply({
+            content: `I can't find the user ${user.username}`,
+            ephemeral: true
+        });
+        interaction.reply({
+            content: `The user ${user.username}#${user.discriminator} has currently **${balanceV['dataValues']['balance']} ${interaction.client.configurations['economy-system']['config']['currencySymbol']}**.`,
+            ephemeral: true
+        });
+    }
+};
+
+module.exports.config = {
+    name: 'economy-system',
+    description: 'general economy-system',
+    defaultPermission: true,
+    options: [
```

Comment:
I recommend exporting a function here and - based on the configuration - not returning the add, remove and set commands at all. 
You can see an example of this [here](https://github.com/SCNetwork/CustomDCBot/blob/687ac39708ad292c931d2cf3bcb3666ea04076ec/modules/levels/commands/manage-levels.js#L75).